youtube_dl/extractor/gramofononline.py

```diff
@@ -0,0 +1,83 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+import re
+import json
+
+
+class GramofonOnlineIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?gramofononline\.hu(/(listen.php\?.*track=)?(?P<id>[0-9]+))?'
```

Comment:
id must be captured correctly in all cases.

Reply:
That's here is just to account for the autogenerated playlist which you get when visiting, https://gramofononline.hu (i.e. in that specific case there is no id).

Reply:
But when using  
https://gramofononline.hu/1401835664/papageno-duett or  
https://gramofononline.hu/listen.php?autoplay=true&track=1401835664  
the id is properly captured